components/google_recaptcha/actions/validate-recaptcha/validate-recaptcha.ts

```diff
@@ -0,0 +1,36 @@
+import googleRecaptcha from "../../app/google_recaptcha.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Validate reCAPTCHA Response",
+  version: "0.0.1",
+  key: "google_recaptcha-validate-recaptcha",
+  description: "Validate a Google reCAPTCHA request (v2 or v3). [See docs here](https://developers.google.com/recaptcha/docs/verify)",
+  props: {
+    googleRecaptcha,
+    remote_ip: {
+      type: "string",
+      label: "Remote IP Address",
+      description: "The IP Address originating the request. This whould be the user IP where the challenge was executed on the frontend.",
+      default: "{{steps.trigger.event.client_ip}}",
```

Comment:
@zalmanlew, the default value for Remote IP Address might be confusing for other users. I suggest we have this optional only (as stated by Google Recaptcha API)
```suggestion
      optional: true,
```

Reply:
@vunguyenhung can it have a default value and be optional? Like this if a user did want to enable it it would already have the export for the IP address filled.